metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "datadog-datadog_agent",
-  "version": "2.9.0",
+  "version": "3.0.0",
```

Comment:
@albertvaka maybe you should add yourself as an author at this point. Incredible work.